Tests/SwiftDocCTests/Indexing/NavigatorIndexTests.swift

```diff
@@ -726,7 +729,7 @@ Root
         XCTAssertEqual(Set(navigatorIndex.languages), Set(["Swift"]))
         
         // Get the Swift language group.
-        XCTAssertEqual(navigatorIndex.navigatorTree.numericIdentifierToNode[1]?.children.count, 5)
+        XCTAssertEqual(navigatorIndex.navigatorTree.numericIdentifierToNode[1]?.children.count, 4)
```

Comment:
Now that the UncuratedClass has no longer fallen out of the navigation hierarchy there is one fewer top-level item.